tests/test_edi_835_files/sample_835_with_interests.txt

```diff
@@ -0,0 +1,167 @@
+TRN*1*081000607720039*1311732404~
+REF*EV*XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX~
+DTM*405*20220105~
+N1*PR*WSFG - BENEFITS~
+N3*400 BROADWAY~
+N4*CINCINNATI*OH*45201~
+REF*2U*31048~
+REF*EO*8593028~
+PER*CX*WESTERN SOUTHERN BENEFITS*TE*5136291100~
+PER*BL*WESTERN SOUTHERN BENEFITS*TE*5136291100~
+PER*IC**UR*FOR DISCOUNT QUESTIONS CALL ZELIS AT 1-888-266-3053. ALL OTHER CLAIM INQUIRIES CALL WESTERN SOUTHERN AT 513-629-1400. FOR DETAILED PAYMENT INFO, SEE THE EOP ON CHANGE HEALTHCARE WEBSITE.~
+N1*PE*UNIVERSITY PRIMARY CARE PRACTICES INC*XX*1720220098~
+N3*PO BOX 772040~
+N4*DETROIT*MI*482772040~
+REF*TJ*341768928~
+REF*PQ*other_payee_id2~
+LX*1~
+TS3**facility_type_code2~
+CLP*patient_control_number2*1*157*132.81*20*15*E000379133-00*11~
+NM1*QC*1*patient_last_name2*patient_first_name2****MI*patient_id2~
+NM1*82*1*FURNESS*PATRICK****XX*1215927314~
+MIA*2*3*4*3*remark_code2*5*6*7*8***************remark_code04~
+MOA*1*2*3*4*****~
+REF*1L*10000000~
+REF*28*121121~
+REF*SY*274866438~
+DTM*050*20220104~
+AMT*AU*152.81~
+SVC*HC>99214*157*132.81**1~
+DTM*472*20211223~
+CAS*PR*3*20~
+CAS*CO*45*4.19~
+REF*0K*MEDICAL MUTUAL SUPERMED PPO NETWORK~
+AMT*B6*152.81~
+SVC*HC>3008F*0*0**0**1~
+DTM*472*20211223~
+SVC*HC>3075F*0*0**0**1~
+DTM*472*20211223~
+SVC*HC>1159F*0*0**0**1~
+DTM*472*20211223~
+SVC*HC>3079F*0*0**0**1~
+DTM*472*20211223~
+CLP*patient_control_number3*2*268*5.3*20*15*M000475749-00*11*1~
+NM1*QC*1*patient_last_name3*patient_first_name3*R***MI*patient_id3~
+NM1*74*1******C*96400662701~
+NM1*82*1*RABER*DOUG****XX*1578554846~
+MIA*2*3*4*3*remark_code3*5*6*7*8***************remark_code04~
+MOA*1*2*3*4*****~
+REF*1L*10000000~
+REF*28*102720~
+REF*SY*277282337~
+DTM*050*20211222~
+AMT*AU*268~
+SVC*HC>99214>25>CR*157*5.3**1~
+DTM*472*20211202~
+CAS*PR*3*20~
+CAS*OA*23*131.7~
+REF*6R*UMP10063279654-1~
+AMT*B6*157~
+SVC*HC>G0008*44*0**1~
+DTM*472*20211202~
+CAS*OA*23*44~
+REF*6R*UMP10063279654-2~
+AMT*B6*44~
+SVC*HC>90662*67*0**1~
+DTM*472*20211202~
+CAS*OA*23*67~
+REF*6R*UMP10063279654-3~
+AMT*B6*67~
+CLP*patient_control_number4*2*157*1.51*20*15*M000478090-00*11*1~
+NM1*QC*1*patient_last_name4*patient_first_name4*E***MI*patient_id4~
+NM1*74*1******C*91900648501~
+NM1*82*1*GIANANTONIO*KAY***CNP*XX*1619346038~
+MIA*2*3*4*3*remark_code4*5*6*7*8****~
+MOA*1*2*3*4*****~
+REF*1L*10000000~
+REF*28*104938~
+REF*SY*299325566~
+DTM*050*20211231~
+AMT*AU*157~
+AMT*I*5.2~
+SVC*HC>99214*157*1.51**1~
+DTM*472*20211216~
+CAS*PR*3*20~
+CAS*OA*23*135.49~
+REF*6R*UMP10063743914-1~
+AMT*B6*157~
+CLP*patient_control_number5*2*157*1.51*20*15*M000478109-00*11*1~
+NM1*QC*1*patient_last_name5*patient_first_name5****MI*patient_id5~
+NM1*74*1**RAYMOND*E~
+NM1*82*1*GIANANTONIO*KAY***CNP*XX*1619346038~
+REF*1L*10000000~
+REF*28*104938~
+REF*SY*299325566~
+DTM*050*20211231~
+AMT*AU*157~
+AMT*I*5.1~
+SVC*HC>99214*157*1.51**1~
+DTM*472*20211216~
+CAS*PR*3*20~
+CAS*OA*23*135.49~
+REF*6R*UMP10063743920-1~
+AMT*B6*157~
+LQ*HE*code2~
+PLB*1001*20211216*L6*-10.3~resnCode>adjID>5***~
```

Comment:
is the `~` character in this line meant to be a newline character or an element separator?

Reply:
I think it was just meant to be an element separator should I move the `resnCode>adjID>5***~` to the next line?